### ISSUE TITLE
Where will the video recordings of this tutorial be available?

### ISSUE THREAD

**Issue by DiegoALeon** [githubteacher/flaming-octo-meme#156]
Where will the video recordings of this tutorial be available?
Will that include, both yesterday's and today's tutorials?

**Comment by githubstudent** [comment]
@DiegoALeon yes, we are recording this session and will send the links out to everyone who registered!
